package.json

```diff
@@ -76,8 +76,8 @@
   },
   "dependencies": {
     "@dotlottie/react-player": "^1.6.3",
-    "@expensify/react-native-live-markdown": "0.1.210",
     "@expensify/react-native-background-task": "file:./modules/background-task",
+    "@expensify/react-native-live-markdown": "0.1.210",
     "@expo/metro-runtime": "^4.0.0",
```

Comment:
can you remove this change? I think it would cause a merge conflict on one active PR